src/Domain/Entities/Inventory.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Domain.Common;
+
+namespace Domain.Entities
+{
+    class Inventory : BaseEntity
+    {
+        public int ProductId { get; set; }
+        public int Quantity { get; set; }
+        public decimal UnitPrice { get; set; }
+        public DateTime AddedOn { get; set; }
```

Comment:
Should this be needed as we have `CreatedAt` in `BaseEntity` @deba1 @kanismim ?

Reply:
> Should this be needed as we have `CreatedAt` in `BaseEntity` @deba1 @kanismim ?

No. We don't need it anymore.